.github/workflows/ci.yml

```
@@ -1479,6 +1484,91 @@ jobs:
 
+    # TEMPORARY DIAGNOSTIC JOB (issue #458): stress the flaky Windows restart test in a tight loop
+    # on a single runner to reproduce the man-current.next IOException and capture the holder via
+    # FileHolderDiagnostics. Not wired into ci-success so it never gates/block the PR; remove after
+    # the flake is diagnosed.
+    stress-458-diagnostics:
+        name: Stress KillDuringCompactionReplaysInsert (Windows diagnostics)
+        runs-on: windows-latest
+        timeout-minutes: 40
+        needs: [ build-windows ]
+        if: |
+            ((github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' || github.ref == 'refs/heads/main' || github.base_ref == 'main' || github.ref == 'refs/heads/develop' || github.base_ref == 'develop')) && (
```

actionlint (1.7.12)

[error] property "changes" is not defined in object type {build-windows: {outputs: {}; result: string}}

(expression)